gym/vector/async_vector_env.py

```diff
@@ -308,12 +306,15 @@ def call_async(self, name, *args, **kwargs):
         ----------
         name : string
             Name of the method or property to call.
+
+        args, kwargs :
+            Arguments and keyword arguments to apply to the method call.
```

Comment:
```suggestion
        args, kwargs :
            Arguments and keyword arguments to apply to the method call.
```

Nitpick: remove extra whitespace.

Reply:
I didn't know what was the change you suggested (where the extra whitespace was), but I changed the docstring to match more closely the [Numpy docstring guidelines](https://numpydoc.readthedocs.io/en/latest/format.html#parameters). I also moved the docstrings from `SyncVectorEnv` to the base class `VectorEnv` to match what was done for the other methods.

Reply:
it's really just a small thing; there was a leading newline (`\n`) above `args, kwargs` that separated it from the previous parameter. If you look closely the suggestion changes 3 lines of code into 2 lines of code.